metadata.yaml

```diff
@@ -28,4 +28,4 @@ requires:
 storage:
   opensearch-data:
     type: filesystem
-    location: /var/snap/opensearch  # /mnt/opensearch/data
+    location: /var/snap/opensearch/common  # /mnt/opensearch/data
```

Comment:
Why wouldn't we want the entire snap directory?

Reply:
because the only part we care about to be persisted / attached / re-mounted etc.. are the directories that contain data / logs, the rest can just be put back by the charm itself